core/frontend/src/FrustumAnimator.ts

```diff
@@ -55,8 +86,9 @@ export class FrustumAnimator implements Animator {
         view3.lookAtViewAlignedVolume(beginRange.union(endRange), viewport.viewRect.aspect); // set up a view that would show both extents
         duration *= zoomSettings.durationFactor; // increase duration so the zooming isn't too fast
         extentBias = view.getExtents().minus(begin.extents); // if the camera is off, the "bias" is the amount the union-ed view is larger than the starting view
-        if (begin.cameraOn)
-          eyeBias = zVec.scaleToLength(zVec.dotProduct(begin3.camera.eye.vectorTo(view3.camera.eye))); // if the camera is on, the bias is the difference in height of the two eye positions
+        // EDL I don't understand what the eye bias is for.  The eventual use in prior versions adds this to eye point (before computing target?)
+        // if (begin.cameraOn)
+        //  _eyeBias = zVec.scaleToLength(zVec.dotProduct(begin3.camera.eye.vectorTo(view3.camera.eye))); // if the camera is on, the bias is the difference in height of the two eye positions
```

Comment:
@EarlinLutz "eyeBias" performs the "google earth" transition that causes the camera to move up when moving between two zoomed in places. It zooms out so that you can see both where you were and where you're going

Reply:
I suggest that you rename this PR to something like "make FrustumAnimator follow smoother eye and target path when camera direction changes" since this change affects all frustum animations, not just view undo/redo.